docs/manifest.json

```diff
@@ -1793,6 +1794,65 @@
                 ]
               ]
             },
+            {
+              "title": "Astro",
+              "collapse": true,
+              "icon": "astro",
+              "items": [
+                [
+                  { "title": "Overview", "href": "/docs/references/astro/overview" },
+                  { 
+                    "title": "UI Frameworks", 
+                    "items": [  
+                      [
+                        {
+                          "title": "React",
```

Comment:
```suggestion
                          "title": "Use Clerk with Astro and React",
```

Reply:
Not sure if this needed, because it's under "UI frameworks". If we add another one in the future we could have a lot of duplication in the leading words.

- "Use Clerk with Astro and React"
- "Use Clerk with Astro and Solidjs"

Reply:
My take is to use `Use Clerk with Astro and React` for now. If we add more we can re-evaluate. This change would improve search results in docs and just makes it super clear what the page is for.

Your concern is valid, but you did say 'If we add another...' so if we're not sure we're adding another yet then let's assume this is the only one. If we do add another its easy to change this.